deeppavlov/models/classifiers/keras_classification_model.py

```diff
@@ -81,9 +79,10 @@ class KerasClassificationModel(KerasModel):
 
     def __init__(self, embedding_size: int, n_classes: int,
                  model_name: str, optimizer: str = "Adam", loss: str = "binary_crossentropy",
-                 learning_rate: float = 0.01, learning_rate_decay: float = 0.,
+                 learning_rate: Union[float, List[float]] = None,
                  last_layer_activation="sigmoid",
                  restore_lr: bool = False,
+                 learning_rate_decay: Optional[Union[float, str]] = 0.,
```

Comment:
If you're adding a new argument, it's best to add it to the end, so that the order is not disturbed.